src/sprite_picture.cpp

```diff
@@ -169,7 +181,9 @@ int Sprite_Picture::GetFrameWidth() const {
 	const auto& data = pic.data;
 
 	auto& bitmap = GetBitmap();
-	assert(bitmap);
+	if (!bitmap) {
```

Comment:
We use ``assert`` for "this should never ever happen during normal execution otherwise it is a bug in the code".
Can you explain how you triggered this "impossible" assert?

Reply:
Playing around with my test project, I can consistently crash the engine whenever I do the following in this order:

1. We modify a X and Y variable
2. We immediately move an anchored picture and set its coordinates to X and Y.

Even if we add an wait period between both commands, it still crashes.
If we invert these steps, it doesn't happen anymore, go figure.

Here's the test project if you want to play around with: 
[PictureTest-assert.zip](https://github.com/EasyRPG/Player/files/13823922/PictureTest-assert.zip)